### ISSUE TITLE
HS200 LED Won't Turn Off

### ISSUE THREAD

**Issue by mozzy88** [tnmendes/watt#10]
Hey, I have the HS200 and I have attempted to turn on night mode to turn off the LED, but it doesn't work. The WiFi symbol flashes, but the LED does not turn off. I attempted this same process on another HS200 that I have, but it didn't work on that one either.

Please let me know if you need any other information.

Thanks for creating this app!

**Comment by tnmendes** [comment]
Hi @mozzy88 
When you press to turn on the night mode you see a message "Fail to toggle switch"?
And the turn On/ off the device also does not work?

I believe that I know what is happening. If its what I'm thinking i believe that some days I can launch a new version with the fix.

By the way what devices you have from TP-Link!?

**Comment by mozzy88** [comment]
No, I do not see the "Fail to toggle switch" message when I turn on night mode. In the app there is no error message when I attempt to turn night mode on. When I turn off night mode, the WiFi indicator on the physical switch turns green, then shuts off after a while.

The Turn On/Off feature works properly.

I have the HS105, HS200, and LB100 devices.

Please let me know if you need anything else :)

Thanks again.

**Comment by tnmendes** [comment]
Hey @mozzy88, 
I release a new version (2.3.1) can you update and check if the error still persists? Thank you

**Comment by mozzy88** [comment]
Hi @tnmendes, I just updated the app on my iPhone and the issue still persists. I tried it on both of my HS200 switches and the LED won't turn off on either of them.

Please let me know if you need any more information.

**Comment by tnmendes** [comment]
Can you send me "Hi" to my email voless.app@gmail.com so we can try other ways.

Thanks

**Comment by tnmendes** [comment]
After several attempts to understand what is happening, I was not able to understand.
When the nightmode is activated, the response of the device is that it was activated without errors but by the feedback from mozzy88 the LEDs on the HS200 it remains the same.
I will leave this issue open for more users with the same problem to report it here.

Thank you

**Comment by tnmendes** [comment]
I sow i post on TP-Link community that was made by the user Axelvigo

https://community.tp-link.com/us/home/forum/topic/165992?replyId=435124

That user made interesting table 


Model | Hardware Version | Firmware Version | Notes
-- | -- | -- | --
HS210 | 1.0 | 1.5.8 | Works - Watt app turns off LED with "Night mode" enabled.
HS220 | 1.0 | 1.5.11 | Works - Watt app turns off LED and dimmer LEDs (when top buttons are pressed) with "Night mode" enabled.
HS200 | 2.0 | 1.5.7 | Works - Watt app turns off LED with "Night mode" enabled.
HS200 | 1.0 | 1.2.5 | Does NOT work. Night mode toggles WiFi indicator light only.
HS105 | 1.0 | 1.5.6 | Works - Watt app turns off blue LED with "Night mode" enabled.

**Comment by mozzy88** [comment]
Thanks for the feedback @tnmendes. I will have to purchase an HS200 (V2.0)

**Comment by Nick3207** [comment]
I  am using hs200 v1 and when I turn on the night mode , the led light is always on even when the switch is turned on.

**Comment by tnmendes** [comment]
@Nick3207 because of some limitations the night mode does not work on HS200 v1.
I believe that limitation is the hardware for the v1 but I am not an expert on this.
